Add Astral Credits (XAC) to token list

HelloMokuzai opened the pull request:
Howdy!

Just looking to add our projects token to your approved token list.

Astral Credits (XAC) is an openzeppelin ERC20 token on the SONGBIRD blockchain.
XAC is distributed freely to the community using a faucet with a built-in deflationary supply schedule.
The project is live and Blazeswap looks to be the DEX of choice for our community.
You can find details regarding our project on our website - https://astralcredits.xyz/

Any questions, let me know!

MKZI⚡️

alexdupre commented:
Thanks for the submission, may I ask you to:
1) verify the contract source code on the explorer
2) use the correct dimensions and padding for the svg file
3) move the tokens.json entry below to keep the sorting by symbol name
to comply with the listing rules, please?

HelloMokuzai commented:
Hi @alexdupre,

I have updated the image file to be the correct spec (500*500px, 8px padding) and moved the tokens.json entry as requested.

Apologies but I'm not sure if I can verify on the explorer. I have the original .sol file but not the fully compiled copy of the source code including original deps. I can use this to rebuild and interact with the contract on remix but unfortunately when flattened and attempt to verify, the bytecode doesn't match.

The contract uses the default openzeppelin ERC20 configurations. Is there anything that can be done? We would really love to use your platform.

alexdupre commented:
Not everything is lost, the compilation is deterministic, so if you use the same compiler version and the same number of optimization runs you'll still be able to verify the contract (assuming the source code has not been altered, including the openzeppelin dependency). If you still have the original source code in remix, it might be a matter to try a few solc versions and a couple of optimization possibilities (none and 200 are the most common alternatives). Let me know how it goes, and upload the flattened file here in case. The issuer of GEMIN had the very same problem and was eventually able to verify it.
